src/InterventionBackend.php

```diff
@@ -386,6 +452,18 @@ protected function getResourceDimensions(InterventionImage $resource)
     }
 
     /**
+     * Cache key for recording errors
+     *
+     * @param string $hash
+     * @param string $variant
+     * @return string
+     */
+    protected function getErrorCacheKey($hash, $variant)
+    {
+        return self::CACHE_MARK . sha1($hash . '-' . $variant);
```

Comment:
`$variant` could potentially be null, but I guess leaving the trailing `-` is fine right? it's just generating a sha1 anyway :D